scripts/stats.ts

```diff
@@ -1,43 +1,82 @@
 import { Compat } from "compute-baseline/browser-compat-data";
+import { fileURLToPath } from "node:url";
+import yargs from "yargs";
 import { features } from "../index.js";
 
-const featureCount = Object.keys(features).length;
-
-const keys = [...new Compat().walk()]
-  .filter((f) => !f.id.startsWith("webextensions"))
-  .map((f) => f.id).length;
-
-const compatKeys = Object.values(features).flatMap(
-  (f) => f.compat_features ?? [],
-).length;
-
-const featureSizes = Object.values(features)
-  .map((feature) => (feature.compat_features ?? []).length)
-  .sort((a, b) => a - b);
-
-const stats = {
-  features: featureCount,
-  compatKeys,
-  compatCoverage: compatKeys / keys,
-  compatKeysPerFeatureMean: compatKeys / featureCount,
-  compatKeysPerFeatureMedian: (() => {
-    const sizes = featureSizes;
-    const middle = Math.floor(sizes.length / 2);
-    return sizes.length % 2
-      ? sizes[middle]
-      : (sizes[middle - 1] + sizes[middle]) / 2;
-  })(),
-  compatKeysPerFeatureMode: (() => {
-    const frequencyMap = new Map<number, number>();
-    for (const size of featureSizes) {
-      frequencyMap.set(size, (frequencyMap.get(size) ?? 0) + 1);
+const argv = yargs(process.argv.slice(2))
+  .scriptName("stats")
+  .usage("$0", "Generate statistics")
+  .option("verbose", {
+    alias: "v",
+    describe: "Show more detailed stats",
+    type: "count",
+    default: 0,
+  }).argv;
+
+export function stats(detailed: boolean = false) {
+  const featureCount = Object.keys(features).length;
+
+  const keys = [];
+  const doneKeys = Object.values(features).flatMap(
+    (f) => f.compat_features ?? [],
+  );
+  const toDoKeys = [];
+  const deprecatedNonStandardKeys = [];
+
+  for (const f of new Compat().walk()) {
+    if (!f.id.startsWith("webextensions")) {
+      keys.push(f.id);
+
+      if (!f.deprecated && f.standard_track) {
+        if (!doneKeys.includes(f.id)) {
+          toDoKeys.push(f.id);
+        }
+      } else {
+        deprecatedNonStandardKeys.push(f.id);
+      }
     }
-    return [...frequencyMap.entries()]
-      .sort(
-        ([sizeA, frequencyA], [sizeB, frequencyB]) => frequencyA - frequencyB,
-      )
-      .pop()[0];
-  })(),
-};
-
-console.log(JSON.stringify(stats, undefined, 2));
+  }
+
+  const featureSizes = Object.values(features)
+    .map((feature) => (feature.compat_features ?? []).length)
+    .sort((a, b) => a - b);
+
+  const result = {
+    features: featureCount,
+    compatKeys: doneKeys.length,
+    compatKeysUnmapped: toDoKeys.length,
```

Comment:
So, there are two numbers for unmapped keys here:
(1) `toDoKeys.length`
(2) `toDoKeys.length` + `deprecatedNonStandardKeys.length`

The current `compatCoverage` percentage takes into account (2) but `compatKeysUnmapped` doesn't. I think this is a bit confusing.

currentBurndown: toDoKeys.length
compatKeysUnmapped: toDoKeys.length + deprecatedNonStandardKeys.length

Reply:
Yes, this is a good idea. I've changed this a bit based on your comment:

- `compatKeys` is what we have in the feature data
- `compatKeysUnmapped` is `toDoKeys.length + deprecatedNonStandardKeys.length`
- `currentBurndownSize` is the number of standard, non-deprecated keys left to map
- `currentBurndown` is the array of standard, non-deprecated keys left to map